infra/dev/docker-compose.yml

```diff
@@ -0,0 +1,77 @@
+version: "3.7"
+services:
+  nginx:
+    image: nginx:stable
+    container_name: dev_nginx
+    ports:
+      - 80:80
+    volumes:
+      - ./nginx/nginx.conf:/etc/nginx/nginx.conf
+    depends_on:
+      - laravel
+    networks:
+      - paul_net
+
+  # Here we delegate the dependency installation to composer container
+  composer:
+    image: composer:latest
+    user: ${USER_ID}:${GROUP_ID}
+    working_dir: /var/www
+    volumes:
+      - ../../src:/var/www
+    command: composer install
+  
+  #NodeJs container will simply compile the package.json 
+  nodejs:
+    image: node:lts
+    user: ${USER_ID}:${GROUP_ID}
+    working_dir: /var/www
+    volumes:
+      - ../../src:/var/www
+    command: npm install
+    depends_on:
+      - composer
+
+  laravel:
+    build:
+      context: ./laravel
+      args:
+        USER_ID: ${USER_ID}
+        GROUP_ID: ${GROUP_ID}
+    container_name: dev_laravel
+    volumes:
+      - ../../src:/var/www
+      - ./laravel/php-fpm.conf:/usr/local/etc/php-fpm.d/zz-docker.conf
+    ports:
+      - 42090:9000
+    networks:
+      - paul_net
+    depends_on:
+      - composer
+      - db
+
+  db:
+    image: postgres:12.2
+    container_name: dev_postgres
+    restart: always
+    ports:
+      - 5432:5432
+    volumes:
+      - ./postgres/initScript:/docker-entrypoint-initdb.d
+    environment:
+      POSTGRES_USER: laravel
+      POSTGRES_PASSWORD: passPRO
+  
+  adminer:
```

Comment:
What is the purpose of this container?

Reply:
it a "php-mysql like", it can be useful to have a web browser to interrogate the db

Reply:
This isn't needed, if we really need to access the db we can use tools like mysql-workbenchlike but for postgres

Reply:
Fixed in 30c050b